src/repo/user.ts

```diff
@@ -24,12 +24,26 @@ export class KnexUserRepository implements UserRepository {
         return typeof identity === 'undefined' ? null : identity;
     }
 
-    private async findUser(userId: string): Promise<User> {
-        return await this.knex
+    public async findUser(userId: string): Promise<User> {
+        const user = await this.knex
             .withSchema('public')
             .first('id', 'created', 'updated', 'default_identity as defaultIdentity')
             .from<User>('user')
             .where('id', userId);
+
+        if (!user) {
+            return user;
+        }
+
+        const identities = await this.knex
```

Comment:
Is the query guaranteed to return an empty array if there are no results?

Reply:
i'll need to check that

Reply:
as far as i can tell, yes